backend/app/api/endpoints/admin/public_models.py

```
@@ -0,0 +1,206 @@
+        model.is_active = model_data.is_active
+
+    db.commit()
+    db.refresh(model)
+
+    return _model_to_response(model)
+
+
+@router.delete("/public-models/{model_id}", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_public_model(
+    model_id: int = Path(..., description="Model ID"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)